http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=643588

Description: Fix for incompatibility with autoconf2.13
 This package will not build if autconf2.13 is installed.
 Adding AC_PREREQ allows the automatic checks to run the correct
 version of autconf, whichever is installed, or to get a clear error
 message about the problem.
 
Author: Wookey <wookey@linaro.org>

--- acl-2.2.49.orig/configure.in
+++ acl-2.2.49/configure.in
@@ -13,6 +13,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
+AC_PREREQ([2.59])
 AC_INIT(include/acl.h)
 AC_CONFIG_AUX_DIR([.])
 AC_CONFIG_MACRO_DIR([m4])
